base/shell/explorer/traywnd.cpp

```diff
@@ -3160,6 +3171,51 @@ class CTrayWindow :
         return (LRESULT)m_TaskSwitch;
     }
 
+    BOOL IsTaskWnd(HWND hWnd)
```

Comment:
Looks like this function is copy-pasted from CTaskSwitchWnd::IsTaskWnd . Can we do better and calling the one from task switch? (or vice-versa)

Reply:
Added a pure function member to interface `ITrayWindow` and implemented.